src/util/byte_operators.h

```diff
@@ -25,7 +25,7 @@ Author: Daniel Kroening, kroening@kroening.com
 /// The object can either be interpreted in big endian or little endian, which
 /// is reflected by the \c id of the expression which is either
 /// \c ID_byte_extract_big_endian or \c ID_byte_extract_little_endian
-class byte_extract_exprt:public binary_exprt
+class byte_extract_exprt : public binary_exprt
```

Comment:
This came up as an error with cpplint, hence the change. I will make a note for future PRs.
Would you suggest I revert this back or just leave it as it is?

Reply:
I am not too concerned either way. The ideal is to only fix the style where the linter CI job in Travis complains. This is because this job only checks the style of the lines which have been updated as part of the PR. So the ideal would be to revert this change if you can do so without the CI job failing. That way when we check the git blame for any given line, we have a better chance of seeing the last functional change. You can do the equivalent check to the CI job, but locally, by checking out your branch and running `./scripts/run_lint.sh develop`, from the root of the repository. You may need to install the python unidiff package first with `pip install --user unidiff`.